core/templates/dev/head/pages/story-viewer-page/chapters-list/story-viewer-chapters-list.directive.ts

```diff
@@ -0,0 +1,231 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the learner's view of a story.
+ */
+
+require(
+  'components/common-layout-directives/common-elements/' +
+  'attribution-guide.directive.ts');
+
+require('components/summary-tile/exploration-summary-tile.directive.ts');
+
+require('domain/story_viewer/StoryPlaythroughObjectFactory.ts');
+require('domain/utilities/UrlInterpolationService.ts');
+require('services/AlertsService.ts');
+require('services/PageTitleService.ts');
+
+oppia.animation('.oppia-story-animate-slide', function() {
+  return {
+    enter: function(element) {
+      element.hide().slideDown();
+    },
+    leave: function(element) {
+      element.slideUp();
+    }
+  };
+});
+
+oppia.directive('storyViewerChaptersList', ['UrlInterpolationService',
+  function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {
+        getPlaythroughObject: '&playthroughObject'
```

Comment:
This should be in `bindToController: {}`

Reply:
Done